components/brave_wallet/browser/brave_wallet_utils_unittest.cc

```diff
@@ -804,6 +834,17 @@ TEST(BraveWalletUtilsUnitTest, GetNetworkURLTest) {
       GURL("https://test-btc.com"),
       GetNetworkURL(&prefs, mojom::kBitcoinMainnet, mojom::CoinType::BTC));
 
+  EXPECT_EQ(GURL("https://mainnet.lightwalletd.com:9067/"),
```

Comment:
We should be proxying mainnet by default. Testnet we have a precedent for not requiring it on so no concerns with that.